src/debugger/debug_workflow.ts

```diff
@@ -173,7 +173,12 @@ export class WorkflowContextDebug extends DBOSContextImpl implements WorkflowCon
         return (check as RecordedResult<R>).output;
       }
       // If we have a proxy, then execute the user's transaction.
-      const result = await txn.call(clsinst, tCtxt, ...args);
+      let cresult: R | undefined;
+      await runWithTransactionContext(tCtxt, async ()=> {
+        cresult = await txn.call(clsinst, tCtxt, ...args);
+      });
+      const result = cresult!;
```

Comment:
So what this does is enforcing the different asserts in `runWithTransactionContext`, right? Because otherwise, we're already calling the `txn` function with the "current" context (`tCtxt`)

Reply:
I may not understand the question.  I want to ensure that the same ctx is available that is passed in as the argument, that's the invariant. tCtxt was made up just there, so it ought to be in place...